tidy3d/__init__.py

```diff
@@ -38,6 +38,7 @@
 # data
 from .components.data.data_array import (
     AxialRatioDataArray,
+    CapacitanceCurveDataArray,
```

Comment:
hmm good point. Right now we're solving direct-current (dc) which, as far as I'm aware, is the only regime clients are interested in. I'm guessing, if we solve for transient or frequency domain we'd have something equivalent and we'd need to generalize. 
So, for the time being, we're not worrying about having frequency dependence.

Reply:
Nice that sounds good!

Reply:
What about we leave it as `CapacitanceDataArray`. It feels generic enough and we can add additional dimensions without modifying the name?

Reply:
Sure! If you have a clear perspective that it'll be an easy frequency domain extension then sure am down for that. Maybe I'm just thinking that different solver configurations should possibly give different data types since that way it'll be much easier to validate and use say in the future for SPICE extraction accoridngly

Reply:
Actually, you might be right. Naming it `DCCapacitanceDataArray` may be the easiest and we can add new data arrays as needed